### ISSUE TITLE
User creatable databases

### ISSUE THREAD

**Issue by jamesgeddes** [GeekZoneHQ/web#46]
Tikiwiki has "[Trackers](https://tikitrackers.org/)" which allows users to create and define a database structure within the site, with no database knowledge required. GZ has already used this to create the [asset register](https://geek.zone/tracker1).

Build a similar - but better - system that allows a user to

- define the data that they want to record
- define the relationships between that data
- record that data

**Comment by CarwynNelson** [comment]
This feels like quite a mammoth task that would require a lot of maintenance going forward. Would something like https://airtable.com/ do this job?

Maybe we could then build an integration with that sort of tool to tie everything together nicely?

**Comment by jamesgeddes** [comment]
Yep, or something like https://baserow.io/ (FOSS alternative) perhaps. Always good to avoid reinventing the wheel.

Would certainly need to be seamlessly available to members while logged into geek.zone

**Comment by CarwynNelson** [comment]
From a quick skim of their website baserow looks great. It would be fantastic if it supported SSO via SAML, but if it doesn't and it's FOSS then maybe it's something we could contribute back to the project.
